pages/api/hunks.ts

```diff
@@ -32,6 +33,8 @@ const hunkHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 	}
 	catch (error) {
 		console.error('[hunkHandler] Error parsing hunk json:', error);
+		const eventProperties = { hunk: hunkJson, response_status: 400 };
```

Comment:
Maybe we should JSON stringify this `hunkJson` object.

Reply:
Discussed a better solution with @MuskanPaliwal :
we will create columns for `repo_provider`, `repo_owner` and `repo_name`, and add the `prhunkvec` value as a JSON string in another column.